install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -110,7 +110,14 @@ spec:
 
 {{- if .Values.nodeinit.reconfigureKubelet }}
                     echo "Changing kubelet configuration to --network-plugin=kubenet"
-                    sed -i "s:--network-plugin=cni\ --cni-bin-dir={{ .Values.cni.binPath }}:--network-plugin=kubenet:g" /etc/default/kubelet
+                    if [[ -f /etc/default/kubelet ]];
+                    then
+                      sed -i "s:--network-plugin=cni\ --cni-bin-dir={{ .Values.cni.binPath }}:--network-plugin=kubenet:g" /etc/default/kubelet
+                    elif [[ -f /etc/systemd/system/kubelet.service.d/10-kubeadm.conf ]];
```

Comment:
Does the presence of `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` mean that we are running on multi-node minikube?